docs/conf.py

```diff
@@ -0,0 +1,35 @@
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath(".."))
+
+from discord.ext.ipc import __version__ as release
```

Comment:
This isn't going to work on rtd. You need to do the same thing that is done in the setup file.
```suggestion
with open("discord/ext/ipc/__init__.py") as stream:
    release = re.search(
        r"^__version__\s*=\s*[\'\"]([^\'\"]*)[\'\"]", stream.read(), re.MULTILINE
    ).group(1)
```